Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/ChallengesStepDefinitions.cs

```diff
@@ -48,36 +48,36 @@ public async Task GivenAChallengeCreatedByAnAnonymousUser(string challengeName)
 
     #region When
 
-    [When("an anonymous user sends a POST request to the /Challenges endpoint")]
+    [When(@"an anonymous user sends a POST request to the \/Challenges endpoint")]
     public async Task WhenAnAnonymousUserSendsAPostRequestIsSentToTheChallengesEndpoint()
     {
         var client = _clientPool.Anonymous;
 
         _responseContext.WhenResponse = _challengeResponse = await client.Challenges.CreateChallengeUnauthenticated();
     }
 
-    [When($"{RegexFor.SINGLE_THING} sends a POST request to the /Challenges endpoint")]
+    [When(@$"{RegexFor.SINGLE_THING} sends a POST request to the \/Challenges endpoint")]
```

Comment:
Why don't you use `{word}` here? There are some other occurrences of this. You should maybe do a global find and replace